jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeTransformUtils.kt

```diff
@@ -127,6 +131,17 @@ fun String.toTransformationLanguage() = when (this) {
     else -> TransformationLanguage.UNKNOWN_TO_SDK_VERSION
 }
 
+fun getJdkVersionText(version: JavaSdkVersion?): String {
+    val jdkVersionText: String = if (CodeModernizerUIConstants.supportedSourceJDKs.contains(version)) { // detected java version is supported
+        "We found Java version: $version. Select a different version if incorrect."
```

Comment:
For the string internationalization, we have to use the `MessageBundle.properties` file to define strings.

You reference those strings like 
```message(<messagebundleId>)``` 

where the `messageBundleId` looks something like -> ```message("codemodernizer.notification.warn.zip_creation_failed.reasons.unknown")```